src/renderer/components/screens/AccountDeletionScreen/AccountDeletionScreen.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useEffect, useState } from 'react'
+import useTranslationFunction from '../../../hooks/useTranslationFunction'
```

Comment:
Mini optional nitpick: I'm trying to keep some sort of grouping of the imports to distinct better where they come from, separated by newlines: external, internal, styles, types

Reply:
sounds like sth a lint could enforce? I agree with this in terms that I like it, but not enough to force myself to follow it as it does not matter much.

Reply:
I've experimented with a linter in a separate branch